### DISCUSSION TITLE
Evaluations not being saved

### DISCUSSION THREAD

**Discussion by AriH2023** [NREL/REopt-API-Analysis#125] [1 - General REopt Questions]
After I run an evaluation in the REopt web tool, while logged in, the evaluation does not show  up in my Saved Evaluations. None of my evaluations have saved. I'm using a Google account to login. How can I save evaluations?

**Comment by amnag** [comment]
I am having exactly the same issue, is there any way to avoid it ?

**Comment by LParkhill** [comment]
A fix was deployed for this issue last night. However, it has been reported that some users are still missing some evaluations in their Saved Evaluations. We continue to debug this issue. Thanks for your patience!
